Basic_viewer/doc/Basic_viewer/Concepts/GraphicsSceneOptions.h

```diff
@@ -0,0 +1,124 @@
+/*!
+\ingroup PkgBasicViewerConcepts
+
+The concept `GraphicsSceneOptions` defines data and methods used to tune the way that the cells of a given data structure of \cgal are considered for drawing or to be added into a graphics scene.
+The different `std::function` can be modified to change for example the behavior of the drawing.
+
+\cgalHasModelsBegin
+\cgalHasModelsBare{\link CGAL::Graphics_scene_options `CGAL::Graphics_scene_options<DS, VertexDescriptor, EdgeDescriptor, FaceDescriptor>`\endlink}
+\cgalHasModelsEnd
+
+*/
+class GraphicsSceneOptions
+{
+public:
+  /*!
+    A data structure of \cgal.
+  */
+  typedef unspecified_type DS;
+
+  /*!
+    A descriptor of vertices of `DS`
+  */
+  typedef unspecified_type vertex_descriptor;
+
+  /*!
+    A descriptor of edges of `DS`
+  */
+  typedef unspecified_type edge_descriptor;
+
+  /*!
+    A descriptor of faces of `DS`
+  */
+ typedef unspecified_type face_descriptor;
+
+  /// `std::function` that returns `true` if the given vertex must be ignored, `false` otherwise.
+  /// Returns `false` by default.
+  std::function<bool(const DS &, vertex_descriptor)> ignore_vertex;
+
+  /// `std::function` that returns `true` if the given edge must be ignored, `false` otherwise.
+  /// Returns `true` by default.
+  std::function<bool(const DS &, edge_descriptor)> ignore_edge;
+
+  /// `std::function` that returns `true` if the given face must be ignored, `false` otherwise.
+  /// Returns `true` by default.
+  std::function<bool(const DS &, face_descriptor)> ignore_face;
+
+  /// `std::function` that returns `true` if the given vertex is colored, `false` otherwise.
+  /// Returns `false` by default.
+  std::function<bool(const DS &, vertex_descriptor)> is_vertex_colored;
+
+  /// `std::function` that returns `true` if the given edge is colored, `false` otherwise.
+  /// Returns `false` by default.
+  std::function<bool(const DS &, edge_descriptor)> is_edge_colored;
+
+  /// `std::function` that returns `true` if the given face is colored, `false` otherwise.
+  /// Returns `false` by default.
+  std::function<bool(const DS &, face_descriptor)> is_face_colored;
+
```

Comment:
If one function in the familly of functions `is_element_colored` returns `false`, what is the displayed color of the elements?

Reply:
Previously discussed here: https://github.com/CGAL/cgal/pull/6967#discussion_r1363589392

Reply:
In such a case, the cell is not colored. Currently, this is the problem of the basic viewer to decide what to do.  
If you think this is a crucial blocking issue, we can add a default color in the option...

Reply:
> In such a case, the cell is not colored. Currently, this is the problem of the basic viewer to decide what to do. If you think this is a crucial blocking issue, we can add a default color in the option...

Maybe you could just document here what are the default colors (that is: the default colors in the basic viewer, if the user does not know who to change it).

Reply:
done.